tests/migrations/test_executor.py

```diff
@@ -658,6 +680,7 @@ def __init__(self, graph, applied):
 
 class FakeMigration:
     """Really all we need is any object with a debug-useful repr."""
+
```

Comment:
Revert changes.

Reply:
This was failing flake8, so I added an empty line.

Reply:
Odd. In all cases we avoid doing unrelated style changes in such feature PR so I'd still revert.